### ISSUE TITLE
False positive with "Unneccessary trailing whitespace" on a simple query

### ISSUE THREAD

**Issue by pwildenhain** [alanmcruickshank/sqlfluff#395]
I'm using the most up-to-date version from `master`

test.sql
```
select 
    field_1
from
    patient
```

```
$ sqlfluff lint test.sql
== [test.sql] FAIL
L:   2 | P:  11 | L001 | Unneccessary trailing whitespace.
```

If I move the field up to the same line as the `select` then it goes away

test.sql
```
select field_1
from
    patient
```

**Comment by pwildenhain** [comment]
The last time I used this package was a couple of months ago, and I didn't have this issue -- installed the package from commit 01f17cc and this issue went away. Not sure how helpful that is, but looks like the regression occurred somewhere within the last couple months

**Comment by pwildenhain** [comment]
NEVERMIND. The trailing whitespace was near the `select` and I didn't notice it

🙇‍♂️ thank you `sqlfluff`
